papers/Sepideh_Alassi_Bertram_Winter/PyCells_for_an_Open_Semiconductor_Industry

```diff
@@ -0,0 +1,270 @@
+:author: Sepideh Alassi
+:email: sepideh.alassi@iis.fraunhofer.de
+:institution: Fraunhofer IIS
+
+:author: Bertram Winter
+:email: Bertram.Winter@ams.com
+:institution: ams AG
+
+
+------------------------------------------------
+PyCells for an Open Semiconductor Industry
+------------------------------------------------
+
+.. class:: abstract
+
+	In modern semiconductor industry, automatic generation of parameterized and recurring layout structures plays an important role and should be present as a feature in Electronic Design Automation (EDA)-tools. Currently these layout generators are developed with a proprietary programming language and can be used with a specific EDA-tool. Therefore, the semiconductor companies find the development of the layout generators that can be used in all state of the art EDA-tools which support OpenAccess database appealing. The goal of this project is to develop computationally efficient layout generators with Python (PyCells), for ams AG technologies, that possess all the features of comprehensive layout generators.
+	
+	
+.. class:: keywords
+
+   PyCells, Semiconductor, OpenAccess
+
+Introduction
+------------
+The number of companies active in modern semiconductor business is increasing every day which raises the demand for cheaper EDA-tools. However, one of the main steps of designing an integrated circuit is drawing the physical structure, the layout. Thus the EDA-tools should have a feature for automatic generation of parametrized, recurring structures as pre-defined layouts in a library. The layout generators which are presently in use, are developed using a proprietary language, SKILL, and are only usable as a feature of a specific EDA-tool. On the other hand, the second major drawback of using the proprietary layout generators is the lack of interoperability characteristic. Consequently, the semiconductor manufacturers are not able to support their customers in an optimal way. Hence, the use of an open standard for an EDA database also allows chip designers to work on the same chip-design using tools, which often have more affordable prices, from different EDA vendors. 
```

Comment:
there are many EDA(Electronic design automation)-tools for circuit design, such as cadence, synopsis, tanner, ...  the purpose of our work was to develop libraries of layout generators for technologies of ams AG, which can be used by different EDA-tools. By interoperability characteristic, we mean that the libraries can be used by various tools without any restriction and designer is free to choose the EDA-tool he/she prefers. Originally all these libraries were developed by SKILL language and could be just used by cadence. The high cost of most of EDA tools was the reason that most of the small companies were not able to afford this tool, and consequently couldn't use the libraries of layout generators.